requirements-docs.txt

```diff
@@ -3,4 +3,13 @@ Sphinx==3.0.2
 sphinx-notfound-page==0.4
 # alabaster 0.7.14 dropped support for Sphinx<3.4
 # and Sphinx 3.0.2 bounds alabaster>=0.7,<0.8.
-alabaster==0.7.13
\ No newline at end of file
+alabaster==0.7.13
+# These packages removed dependency on Sphinx>5,
```

Comment:
Nit: Update the comment to be: `Latest versions of these packages support only Sphix>5`. The packages removed the explicit dependency in the `pyproject.toml` but they still require it at runtime of the extension.